content/product-engineering/engineering/career-development/framework.md

```diff
@@ -398,63 +401,37 @@ To learn more, see ["Considerations for promotion" in our talent review process]
     </tr>
     <!-- IC5 -->
     <tr>
-      <th id="ic5" scope="row" rowspan="3" class="level"><a class="anchor" href="#ic5"></a><abbr title="Individual Contributor">IC</abbr>5</th>
+      <th id="ic5" scope="row" rowspan="2" class="level"><a class="anchor" href="#ic5"></a><abbr title="Individual Contributor">IC</abbr>5</th>
       <td colspan="3" class="level-summary">
-        A different IC role from levels 1-4, with a focus on technical leadership.<br>
-        This level is equivalent in impact to an <a href="../roles#engineering-manager">Engineering Manager</a>.<br>
-        What impact at this level looks like may vary more between individuals than at preceding levels.<br>
+        IC5 is a different role from levels 1-4.<br>
+        What impact looks like at this level varies between individuals.<br>
         We recommend <a href="https://drive.google.com/drive/folders/1fnw5gmlLDDZb9wDodY3KfcTaM3OOQL7G" target="_blank" rel="noopener noreferrer">our company’s copy of “Staff Engineer” by Will Larson</a> as reading material.
```

Comment:
Having a reference to the Staff Engineer book here might be misleading in our attempt not to box all these roles in as "staff"-type roles